Zend/tests/gh10168/assign.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+GH-10168: Assign
+--FILE--
+<?php
+
+class Test {
+    function __destruct() {
+        $GLOBALS['a'] = null;
```

Comment:
in each `__destruct` I would suggest to add `echo "D\n";` at the start to make sure the destruction is really called at the time when expected/needed to test